BEIMA.Client/src/pages/Devices/DevicePage.js

```diff
@@ -171,11 +170,13 @@ const DevicePage = () => {
     const [long, setLong] = useState(device.location.longitude)
     const [locNotes, setLocNotes] = useState(device.location.notes)
 
-    const [imageCopy, setImageCopy] = useState(image)
+    const [newImage, setNewImage] = useState()
+    const [imageCopyUrl, setImageCopyUrl] = useState(image.fileUrl)
```

Comment:
I would change this from imagecopy to something like previewimage... just because its set by both the device image as well as the upload photo input.

Reply:
Good idea

- [ ] change name of image url variable to something more clear

Reply:
Changed variable name to imagePreviewUrl